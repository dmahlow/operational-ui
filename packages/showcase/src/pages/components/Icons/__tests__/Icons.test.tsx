import * as React from "react"
import { render } from "enzyme"

import IconsPage from "../Icons"

describe("Icons Page", () => {
  it("Should render correctly", () => {
    expect(render(<IconsPage />)).toMatchSnapshot()
  })
})
