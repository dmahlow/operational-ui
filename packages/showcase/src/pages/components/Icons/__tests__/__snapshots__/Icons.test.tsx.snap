// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons Page Should render correctly 1`] = `
<div>
  <div
    class="css-by30gj"
  >
    Icons
  </div>
  <p>
    Contiamo's SVG icon set as a single component. It abstracts over different types of icons (react-feather, custom shapes, SVG sprites) to provide a consistent API as the icon set evolves.
  </p>
  <h4>
    Usage
  </h4>
  <div
    class="playground"
  >
    <div
      class="playgroundCode"
    >
      <div
        class="ReactCodeMirror playgroundStage"
      >
        <textarea
          autocomplete="off"
        >
          t]
        </textarea>
      </div>
    </div>
    <div
      class="playgroundPreview"
    >
      <div>
        <div
          class="previewArea"
        />
      </div>
    </div>
  </div>
  <h4>
    Props
  </h4>
  <table
    class="css-81lp1j"
  >
    <thead>
      <tr>
        <th>
          Name
        </th>
        <th>
          Description
        </th>
        <th>
          Default
        </th>
        <th>
          Type
        </th>
        <th>
          Optional?
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          name
        </td>
        <td>
          Icon name, see https://feathericons.com/ (convert name to PascalCase).
        </td>
        <td>
          Play
        </td>
        <td>
          string
        </td>
        <td>
          ðŸš«
        </td>
      </tr>
      <tr>
        <td>
          size
        </td>
        <td>
          Size as pre-defined strings.
        </td>
        <td>
          medium
        </td>
        <td>
          string
        </td>
        <td>
          âœ…
        </td>
      </tr>
      <tr>
        <td>
          sizeOverride
        </td>
        <td>
          Override the size to a single number.
        </td>
        <td />
        <td>
          number
        </td>
        <td>
          âœ…
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
